src/wazuh_modules/wm_command.c

```diff
@@ -251,10 +265,21 @@ cJSON *wm_command_dump(const wm_command_t * command) {
 
 
 // Destroy data
-
+#ifdef WIN32
+DWORD WINAPI wm_command_destroy(void *command) {
+    wm_command_t * ptr_command = (wm_command_t *)command;
+
+    free(ptr_command->tag);
+    free(ptr_command->command);
+    free(ptr_command->full_command);
+    free(ptr_command);
+    return 0;
```

Comment:
this could be summarized with a #ifdef WIN32 on the return to avoid the entire duplication of the function.